### DISCUSSION TITLE
blog/come-sono-finito-alla-nasa

### DISCUSSION THREAD

**Discussion by giscus** [peppocola/pompelmo#2] [Q&A]
# blog/come-sono-finito-alla-nasa

Un mistero più grande di quello di Medjugorje. Storia di come sono finito a collaborare con la NASA.

https://www.pompelmo.xyz/blog/come-sono-finito-alla-nasa

**Comment by giuspillo** [comment]
Daje tutta Peppo ❤️

**Reply by peppocola** [reply]
❤️❤️❤️

**Comment by michelameliddo** [comment]
Sii che bello, bravo peppe ❤️❤️

**Reply by peppocola** [reply]
❤️❤️❤️🙏🏼 grazie
